AdminWebsite/AdminWebsite/ClientApp/src/styles.css

```diff
@@ -143,18 +142,12 @@
     float: left;
 }
 
+.vh-created-booking {
+    background-color: #3868c2;
```

Comment:
use `govuk-colour("<colour name>")` instead. different versions have slightly changed the colours. If you use the function you simplify maintenance by not having to check the colour

Reply:
I used the hex code to avoid changing the file type to scss, and keep the component dependency minimal, but I actually I suppose it doent matter too much. Ill update it